### ISSUE TITLE
Language Server Process Didn’t Start

### ISSUE THREAD

**Issue by aksarben** [PowerShell/vscode-powershell#4694]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [X] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

I just de-installed Visual Studio Code, downloaded version 1.81.0 from https://code.visualstudio.com/download, and installed it.

When I launched , I immediately got an error message: ­

![image](https://github.com/PowerShell/vscode-powershell/assets/6260602/a7b345c9-b7e6-4ca3-806f-7d28bab6d127)

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.6
PSEdition                      Core
GitCommitId                    7.3.6
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Visual Studio Code Version

```console
1.81.1
6c3e3dba23e8fadc360aed75ce363ba185c49794
x64
```

### Extension Version

```console
ms-vscode.powershell@2023.6.0
```

### Steps to Reproduce

Launched the program by clicking the shortcut installed on the desktop.

### Visuals

![image](https://github.com/PowerShell/vscode-powershell/assets/6260602/7a895934-06be-443b-a2a9-34e2b80fc756)

### Logs

The log directory specified in the instructions does not exist, and I could not locate the files. So, originally I pressed Ctrl-Shift-P and pasted the log output here. However, the output was 30 pages long, so I deleted it after I discovered the log locations and pasted links to the individual log files.

**Comment by aksarben** [comment]
Here's the actual location of the log directory (different than what the instructions said:

8/10/2023 3:33:58 PM [VERBOSE] - Creating log directory at: 'file:///c%3A/Users/Dick/AppData/Roaming/Code/User/globalStorage/ms-vscode.powershell/logs/1691699249-47432b06-ba74-40ce-ac79-707ffb8866ad1691699248564'

When I tried to run the debugger, I get these errors:

![image](https://github.com/PowerShell/vscode-powershell/assets/6260602/664d19f4-c6be-4066-9cf2-ba5ce83cf399)

![image](https://github.com/PowerShell/vscode-powershell/assets/6260602/42a671df-926c-432b-bff8-13af982d0d9d)

**Comment by andyleejordan** [comment]
Can you please separate those log files? I can't tell where one ends and another starts. It looks like at some point you change some setting and allow the prompt to restart the session, and after that the status says 'running' so what's the actual state after that?

**Comment by aksarben** [comment]
Hope this works. Would have done this the first time, but the instructions gave the wrong location for the log files, & I couldn't find them. 

[EditorServices.log](https://github.com/PowerShell/vscode-powershell/files/12317706/EditorServices.log)
[StartEditorServices.log](https://github.com/PowerShell/vscode-powershell/files/12317707/StartEditorServices.log)
[vscode-powershell.log](https://github.com/PowerShell/vscode-powershell/files/12317708/vscode-powershell.log)

Looks like it it worked (I think). Do you need anything else?

**Comment by aksarben** [comment]
Looks like I'm not getting notifications of comments/changes to this thread. What e-mail address are they being sent to?

**Comment by andyleejordan** [comment]
I don't know what email addresses you gave GitHub 😂

**Comment by aksarben** [comment]
After a lot of digging, I think I found the right place for notifications settings. Looks like you have to explicitly ask to be notified of comments on issues you’re subscribed to. Seems to defeat the purpose of subscribing. Oh well…

**Comment by aksarben** [comment]
Here's the latest screen shot from when I tried to debug this morning. Still dead in the water:

![image](https://github.com/PowerShell/vscode-powershell/assets/6260602/702aa717-db66-4cda-97d8-c714b379a270)

**Comment by aksarben** [comment]
I’m going through the troubleshooting page again to see if I missed anything. Re the section on **Language Server Protocol Payload Logs**, I’d be be happy to provide these logs if I can. The page says:

**Add the following setting to your settings file: powershell editor services.trace.server:verbose"**

I’m pretty new to VSC, so I’ve only dealt with settings on the settings page in the IDE. What's the name and location this "settings file"?

Also, what folder are the "payload logs" in, and what are the log files named?

**Comment by aksarben** [comment]
Tried to open the the logs folder from Ctrl-Shift-P, but got this error:

![image](https://github.com/PowerShell/vscode-powershell/assets/6260602/5c683342-54a4-46b4-9729-17b22daaf33c)

**Comment by aksarben** [comment]
Ok, finally found how to add "powershell editor services.trace.server:verbose" to my settings. Saved it and re-launched VSC, and got the usual error messages. But when I look in the log directory, none of the log files have changed since yesterday. So either I did something wrong, or the Language Server Protocol Payload Log is located somewhere else?

**Comment by starball5** [comment]
Related on Stack Overflow: [Where can I find the settings file referred to in the VS Code PowerShell extension's troubleshooting page?](https://stackoverflow.com/q/76886852/11107541)

**Comment by aksarben** [comment]
Looks like we can call off the dogs.

When I started VSC yesterday morning, I set the log level to Diagnostic to see if could capture anything helpful in the log.

Lo, and behold, VSC ran flawlessly without a single error!

Don't have a clue why it suddenly started behaving, or what caused the problem in the first place. And, as someone wisely said, "Problems that go away by themselves tend to come back themselves." So I'm keeping my fingers crossed.
